test/unit/ssh-config.test.ts

```diff
@@ -68,7 +95,23 @@ describe('SSHConfig', function() {
 
     for (const host of ['foo', 'foo.bar', 'baz ham']) {
       assert.deepEqual(config.compute(host), {
-        Host: ['foo', '*.bar', 'baz ham'],
+        Host: [
+          {
+            'quoted': false,
+            'separator': ' ',
+            'val': 'foo',
+          },
+          {
+            'quoted': true,
+            'separator': ' ',
+            'val': '*.bar',
+          },
+          {
+            'quoted': true,
+            'separator': ' ',
+            'val': 'baz ham',
+          },
+        ],
```

Comment:
@cyjake This is breaking api typings no? `compute` should return `string | string[]`

Reply:
indeed, compute() result doesn't need to include original config format, I'll try to fix this issue later. thanks for reporting!